### ISSUE TITLE
Closing symbol '}' is missing

### ISSUE THREAD

**Issue by akiwamot** [YangModels/yang#593]
https://github.com/YangModels/yang/blob/b7868ebd6837790079ec2fad51c581f32a13981c/vendor/cisco/xe/16111/Cisco-IOS-XE-atm.yang#L48

**Comment by akiwamot** [comment]
```
revision 2019-03-19{                                                           
      description                                                                 
        "Revision date correction";
```

**Comment by einarnn** [comment]
Resolved in #595.
